kafka-rest/src/main/java/io/confluent/kafkarest/controllers/ReplicaManagerImpl.java

```diff
@@ -84,15 +88,17 @@ public CompletableFuture<List<PartitionReplica>> searchReplicasByBrokerId(
               return KafkaFutures.toCompletableFuture(result.values().get(brokerId));
             })
         .thenCompose(
-            logDirs ->
-                CompletableFutures.allAsList(
-                    logDirs.values().stream()
-                        .flatMap(logDir -> logDir.replicaInfos.keySet().stream())
-                        .map(
-                            partition ->
-                                getReplica(
-                                    clusterId, partition.topic(), partition.partition(), brokerId))
-                        .collect(Collectors.toList())))
+            logDirs -> {
+              logDirs.forEach((key, value) -> log.debug("Describe log Dirs" + key + value));
```

Comment:
I wonder if we can make this a bit more efficient - we loop through logDirs every time, regardless of whether we are running at debug level.

It probably doesn't matter too much, as this is a slow-ish command anyway, but I wonder if 
 log.debug("Describe log dirs {} ", logDirs) is smart enough to print out the content of the Map.

Reply:
Yes, it's smart go print out the map

This is part of the result
```
DEBUG Describe log dirs {/tmp/kafka-logs/server0=(error=NONE, replicas={tst-53=(size=0, offsetLag=-1, isFuture=false), tst-86=(size=0, offsetLag=-1, isFuture=false), tst-20=(size=0, offsetLag=-1, isFuture=false), test-32=(size=0, offsetLag=0, isFuture=false), tst-111=(size=0, offsetLag=-1, isFuture=false), test-98=(size=0, offsetLag=0, isFuture=false), tst-45=(size=0, offsetLag=-1, isFuture=false), tst-78=(size=0, offsetLag=-1, isFuture=false), tst-12=(size=0, offsetLag=-1, isFuture=false), test-24=(size=0, offsetLag=0, isFuture=false), tst-103=(size=0, offsetLag=-1, isFuture=false), test-90=(size=0, offsetLag=0, isFuture=false), tst-37=(size=0, offsetLag=-1, isFuture=false), tst-70=(size=0, offsetLag=-1, isFuture=false), tst-4=(size=0, offsetLag=-1, isFuture=false), test-48=(size=0, offsetLag=0, isFuture=false), tst-95=(size=0, offsetLag=-1, isFuture=false), test-114=(size=0, offsetLag=0, isFuture=false), tst-29=(size=0, offsetLag=-1, isFuture=false), tst-62=(size=0, offsetLag=-1, isFuture=false), test-40=(size=0, offsetLag=0, isFuture=false), tst-87=(size=0, offsetLag=-1, isFuture=false), test-106=(size=0, offsetLag=0, isFuture=false), tst-120=(size=0, offsetLag=-1, isFuture=false), tst-21=(size=0, offsetLag=-1, isFuture=false), tst-54=(size=0, offsetLag=-1, isFuture=false), test-64=(size=0, offsetLag=0, isFuture=false),
```